### PR TITLE
lifecycle: update text to current versioning scheme

### LINKED ISSUE #872: revamp "2016-01-15-lifecycle"
Are these some trivially obsolete sentences?

https://github.com/bitcoin-core/bitcoincore.org/blob/06e171acaf258dd1ad6c4963988931ab6500289e/_posts/en/pages/2016-01-15-lifecycle.md#L29

https://github.com/bitcoin-core/bitcoincore.org/blob/06e171acaf258dd1ad6c4963988931ab6500289e/_posts/en/pages/2016-01-15-lifecycle.md#L33

https://github.com/bitcoin-core/bitcoincore.org/blob/06e171acaf258dd1ad6c4963988931ab6500289e/_posts/en/pages/2016-01-15-lifecycle.md#L38-L39

https://github.com/bitcoin-core/bitcoincore.org/blob/06e171acaf258dd1ad6c4963988931ab6500289e/_posts/en/pages/2016-01-15-lifecycle.md#L81

If y'all keep insisting on [absurdly long lines in tracked source docs][0],
you make maintenance unnecessarily ugly.
Scroll vertically, not horizontally.

[0]: https://github.com/mempool/mempool/pull/1473#discussion_r848912178

### PR DESCRIPTION
**Pull request by fanquake**
Update the text and examples in the lifecycle page to use the current versioning scheme.
Closes #872.

### CODE REVIEW

**Review by MarcoFalke — APPROVED** [review]
lgtm

**Comment by harding** [comment]
I like @MarcoFalke's [suggestion](https://github.com/bitcoin-core/bitcoincore.org/pull/912#discussion_r1021451443) but it LGTM with or without that change.

File: _posts/en/pages/2016-01-15-lifecycle.md [hunk 1]
```diff
@@ -14,38 +14,38 @@ This document describes the life-cycle of the Bitcoin Core software package rele
...
 Once software has reached the "Maintenance End" period it will only receive critical security fixes until the EOL date.
 After EOL, users must upgrade to a later version to receive security updates, even though the community may provide fixes for critical issues on a best effort basis.
 Generally, it is recommended to run the latest maintenance release (point release) of the current or previous major version.
 
 Please note that minor versions get bugfixes, translation updates, and soft forks. Translation on [Transifex][bitcoin-transifex-link] is only open for the last two major releases.
 
-For example, major version 0.9 was released on 2014-03-19 and we provided maintenance fixes (point releases) until 2015-06-16.
-Critical security issues would still be continued to be fixed until the End-Of-Life "EOL" date of 2016-02-28.
+For example, major version 0.21 was released on 2021-01-15 and we provided maintenance fixes (point releases) until 2022-04-25.
+Critical security issues would still be continued to be fixed until the End-Of-Life "EOL" date of 2022-10-01.
```
[the remark this answers is not in the document]
Comment (fanquake) [reply _posts/en/pages/2016-01-15-lifecycle.md]: I've switched to using 22.0 here.

**Review by MarcoFalke — APPROVED** [review]

File: _posts/en/pages/2016-01-15-lifecycle.md [hunk 2]
```diff
@@ -14,38 +14,38 @@ This document describes the life-cycle of the Bitcoin Core software package rele
...
 
-Proposals to change consensus rules are always shipped first in maintenance versions such as 0.11.2, 0.12.1 etc. This makes it easier for enterprise users to assess and test the proposal because of its smaller changeset compared to a major release. It also allows users who follow a more conservative upgrade path to adopt consensus rule changes in a more timely manner.
+Proposals to change consensus rules are always shipped first in maintenance versions such as 22.2, 23.1 etc. This makes it easier for enterprise users to assess and test the proposal because of its smaller changeset compared to a major release. It also allows users who follow a more conservative upgrade path to adopt consensus rule changes in a more timely manner.
 
 ## Maintenance period
 
 We maintain the major versions until their "Maintenance End". We generally maintain the current and previous major release.
-So if the current release is 0.13, then 0.12 is also considered maintained.
-Once 0.14 is released, then 0.12 would be considered at its "Maintenance End".
+So if the current release is 23.0, then 22.0 is also considered maintained.
```
Comment (stickies-v) [inline _posts/en/pages/2016-01-15-lifecycle.md:38]:
```suggestion
For example, if the current release is 23.0, then 22.0 is also considered maintained.
```

File: _posts/en/pages/2016-01-15-lifecycle.md [hunk 3]
```diff
@@ -14,38 +14,38 @@ This document describes the life-cycle of the Bitcoin Core software package rele
...
+Once 24.0 is released, then 22.0 would be considered at its "Maintenance End".
 The older the major release, the more critical issues have to be to get backported to it, and the more to warrant a new minor release.
```
Comment (stickies-v) [inline _posts/en/pages/2016-01-15-lifecycle.md:40]: This phrase is a bit unclear to me: is this a better suggestion, or am I misunderstanding?
```suggestion
The older the major release, the more critical issues have to be to get backported to it, and the more a new minor release is warranted.
```

File: _posts/en/pages/2016-01-15-lifecycle.md [hunk 4]
```diff
@@ -14,38 +14,38 @@ This document describes the life-cycle of the Bitcoin Core software package rele
...
 Once software has reached the "Maintenance End" period it will only receive critical security fixes until the EOL date.
 After EOL, users must upgrade to a later version to receive security updates, even though the community may provide fixes for critical issues on a best effort basis.
 Generally, it is recommended to run the latest maintenance release (point release) of the current or previous major version.
 
 Please note that minor versions get bugfixes, translation updates, and soft forks. Translation on [Transifex][bitcoin-transifex-link] is only open for the last two major releases.
 
-For example, major version 0.9 was released on 2014-03-19 and we provided maintenance fixes (point releases) until 2015-06-16.
-Critical security issues would still be continued to be fixed until the End-Of-Life "EOL" date of 2016-02-28.
+For example, major version 22.0 was released on 2021-09-13 and we provided maintenance fixes (point releases) until 2022-11-15.
```
Comment (stickies-v) [inline _posts/en/pages/2016-01-15-lifecycle.md:47]: This should probably be updated simultaneously with https://github.com/bitcoin-core/bitcoincore.org/blob/master/_includes/posts/maintenance-table.md?plain=1 ?

**Review by stickies-v — APPROVED** [review]
ACK b1fb03dde6dc0c628b3b68f49254ff405b216986 with a couple of improvement suggestions.

**Comment by katesalazar** [comment]
Of course! This was mentioned somewhere, I'm sure,
but then everybody forgot to push it forward.

File: _posts/en/pages/2016-01-15-lifecycle.md (hunk 4 above)
Comment (fanquake) [reply _posts/en/pages/2016-01-15-lifecycle.md:47]: These dates aren't exact, and can be updated, if need be, when the table is updated.

File: _posts/en/pages/2016-01-15-lifecycle.md (hunk 2 above)
Comment (fanquake) [reply _posts/en/pages/2016-01-15-lifecycle.md:38]: Added.

File: _posts/en/pages/2016-01-15-lifecycle.md (hunk 3 above)
Comment (stickies-v) [reply _posts/en/pages/2016-01-15-lifecycle.md:40]: Okay. I still think it's a pretty confusing phrase. I would intuitively interpret this as new minor releases being _more_ warranted/frequent for older major releases, which is pretty much the opposite. Out of scope, so feel free to ignore, but I think this says the same thing in more simple and clear terms.

```suggestion
As a major release becomes older, issues have to be increasingly critical to get backported to it, and an increasing amount or severity of issues is required to warrant a new minor release.
```

Comment (fanquake) [reply _posts/en/pages/2016-01-15-lifecycle.md]: Ok. Have taken the newer wording.

**Pull request merged**